src/core/dev_api/openvino/op/group_query_attention.hpp

```diff
@@ -38,13 +38,17 @@ class OPENVINO_API GroupQueryAttention : public Op {
     bool get_rotary_interleaved() const {
         return m_rotary_interleaved;
     }
+    bool get_is_static_input() const {
+        return m_is_static_input;
+    }
```

Comment:
This getter should be removed as there is 
There is no such attribute, make this check in transformations where required.

Reply:
Deleted